client/state/posts/likes/test/reducer.js

```diff
@@ -283,99 +281,70 @@ describe( 'reducer', () => {
 
 		describe( 'a POST_LIKE action', () => {
 			test( 'should create a new state item if none is found', () => {
-				const state = items( deepFreeze( {} ), like( 1, 1 ) );
+				const state = itemReducer( DEFAULT_ITEM_STATE, like( 1, 1 ) );
```

Comment:
I'm not confident here, but shouldn't we be passing in `undefined` as the first arg to verify default is being set properly:

`const state = itemReducer( undefined, like(1 , 1 ) );`

Reply:
hmmmmm. good call.